server-spi-private/src/main/java/org/keycloak/authentication/RequiredActionFactory.java

```diff
@@ -33,6 +38,19 @@
  */
 public interface RequiredActionFactory extends ProviderFactory<RequiredActionProvider> {
 
+    List<ProviderConfigProperty> MAX_AUTH_AGE_CONFIG_PROPERTIES = ProviderConfigurationBuilder.create()
+            .property()
+            .name(Constants.MAX_AUTH_AGE_KEY)
+            .label("Maximum Age of Authentication")
+            .helpText("Configures the duration in seconds this action can be used after the last authentication before the user is required to re-authenticate. " + //
```

Comment:
It seems to me that the last sentence of  `helpText` about password policy should be removed. As AFAIK with your PR, the password policy is considered only for `UpdatePassword` required action as it was before.